variables.tf

```diff
@@ -51,6 +51,12 @@ variable "task_memory" {
   default     = 512
 }
 
+variable "ephemeral_storage" {
+  type = number
+  description = "(Optional) The amount(in GiB) of ephemeral storage to allocate for the task"
+  default = 0
```

Comment:
Hey @zapion Thank you for the PR and adding an option , could you please run a `terraform fmt` and push the changes?

Reply:
Hey thanks! I just pushed with the modified commit.